ui/src/app/base/components/FormikField/FormikField.js

```diff
@@ -12,15 +12,29 @@ const FormikField = ({
   ...props
 }) => {
   const id = useRef(uuidv4());
+  const {
+    errors,
+    handleBlur,
+    handleChange,
+    setFieldTouched,
+    status = {},
+    touched
+  } = formikProps;
+  const { serverErrors = {}, invalidValues = {} } = status;
   return (
     <Component
-      error={formikProps.touched[fieldKey] && formikProps.errors[fieldKey]}
+      error={
+        (touched[fieldKey] &&
+          (errors[fieldKey] ||
+            (value === invalidValues[fieldKey] && serverErrors[fieldKey]))) ||
```

Comment:
This conditional is a bit hard to parse. Is it possible to split it out into something readable like:
``` javascript
error={isTouched && (hasFormikErrors || hasServerErrors)}
```
?

Reply:
Thanks, updated.